pkg/nsx/services/common/types.go

```diff
@@ -34,6 +34,9 @@ const (
 	TagScopeNCPVNETInterface        string = "ncp/vnet_interface"
 	TagScopeVPCCRName               string = "nsx-op/vpc_cr_name"
 	TagScopeVPCCRUID                string = "nsx-op/vpc_cr_uid"
+	TagValueGroupScope              string = "scope"
+	TagValueGroupSrc                string = "source"
+	TagValueGroupDst                string = "destination"
```

Comment:
I prefer to `TagValueGroupSource` and `TagValueGroupDestination`.

Reply:
Fixed.